app/views/metadata_presenter/resume/return.html.erb

```diff
@@ -1,10 +1,13 @@
 <div class="fb-main-grid-wrapper">
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-    <h1 id="page-heading" class="govuk-heading-xl"><%= t('presenter.save_and_return.resume.heading', service_name: get_service_name) %></h1>
-    <p><%= t('presenter.save_and_return.resume.content') %></p>
+
     <%= form_for @resume_form do |f| %>
       <%= f.govuk_error_summary(t('presenter.errors.summary_heading')) %>
+
+      <h1 id="page-heading" class="govuk-heading-xl"><%= t('presenter.save_and_return.resume.heading', service_name: get_service_name) %></h1>
```

Comment:
Moved the h1 to conform to the design system guidelines for error summary:
`Put the error summary at the top of the main container. If your page includes breadcrumbs or a back link, place it below these, but above the <h1>.`